migration-0.55/main.go

```diff
@@ -0,0 +1,366 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	clickhouse "github.com/ClickHouse/clickhouse-go/v2"
+	"go.uber.org/zap"
+)
+
+type LogField struct {
+	Name      string `json:"name" ch:"name"`
+	DataType  string `json:"dataType" ch:"datatype"`
+	Type      string `json:"type"`
+	IndexType string `json:"indexType"`
+}
+
+type ShowCreateTableStatement struct {
+	Statement string `json:"statement" ch:"statement"`
+}
+
+const (
+	attribute = "attribute"
```

Comment:
This should be `attributes` instead of `attribute` as I remember from the main codebase otherwise `hasMaterializedColumn` would be incorrect for attributes.

Reply:
We added `s` for the keyName check https://github.com/SigNoz/signoz-db-migrations/blob/6b93fddc303051908e7e9cbf6b137832ff00ffb2/migration-0.55/main.go#L175